<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignerWindow</class>
 <widget class="QMainWindow" name="DesignerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoles">
    <property name="title">
     <string>Roles</string>
    </property>
    <addaction name="actionSwitch_Role"/>
   </widget>
   <widget class="QMenu" name="menuCatalog">
    <property name="title">
     <string>Catalog</string>
    </property>
    <addaction name="actionAdd_Catalog"/>
    <addaction name="actionModify_Catalog"/>
   </widget>
   <addaction name="menuRoles"/>
   <addaction name="menuCatalog"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSwitch_Role">
   <property name="text">
    <string>Switch Role</string>
   </property>
  </action>
  <action name="actionAdd_Catalog">
   <property name="text">
    <string>Add Catalog</string>
   </property>
  </action>
  <action name="actionModify_Catalog">
   <property name="text">
    <string>Modify Catalog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
